backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -195,6 +195,7 @@ private async void OnIsarRobotInfo(object? sender, MqttReceivedArgs mqttArgs)
             }
         }
 
+
```

Comment:
Is this something the formatter introduced?

Reply:
Done